packages/cardpay-sdk/contracts/abi/account-registration-nft.ts

```diff
@@ -1,3 +1,6 @@
+/**
```

Comment:
The docs get very polluted with too many variables / classes so I had to add the @category tag to organise them